libs/community/langchain_community/tools/tavily_search/tool.py

```diff
@@ -29,7 +29,14 @@ class TavilySearchResults(BaseTool):
     )
     api_wrapper: TavilySearchAPIWrapper = Field(default_factory=TavilySearchAPIWrapper)  # type: ignore[arg-type]
     max_results: int = 5
+    search_depth: Optional[str] = "advanced"
+    include_domains: Optional[List[str]] = []
+    exclude_domains: Optional[List[str]] = []
+    include_answer: Optional[bool] = False
+    include_raw_content: Optional[bool] = False
+    include_images: Optional[bool] = False
```

Comment:
dont need to be Optional

Reply:
also would be nice to add docstrings for all these (can be in a follow up pr)

Reply:
Should be fixed in commit I just pushed